src/common/feedGenerator.ts

```diff
@@ -125,10 +127,11 @@ export const feedToFilters = async (
           .getRepository(SourceMember)
           .createQueryBuilder('sm')
           .select('sm."sourceId"')
-          .where('sm."userId" = :userId', { userId })
-          .andWhere(
-            `COALESCE((flags->'hideFeedPosts')::boolean, FALSE) = FALSE`,
```

Comment:
So now we pull all source member rows for each user and then manually hide depending on the flag in below iteration?

Reply:
yes. we need to block them, not only remove them from the list

Reply:
Yeah, figured from here https://github.com/dailydotdev/daily-api/pull/1865#issuecomment-2085038766, tnx for explanation!